examples/deepspeed/cifar10_cpu_offloading/zero_3_cpu_offload.yaml

```diff
@@ -14,7 +14,7 @@ environment:
     # You may need to modify this to match your network configuration.
     - NCCL_SOCKET_IFNAME=ens,eth,ib
   image:
-    gpu: determinedai/environments:cuda-11.3-pytorch-1.10-deepspeed-0.8.3-gpu-6eceaca
+    gpu: determinedai/environments:cuda-11.3-pytorch-1.10-deepspeed-0.8.3-gpu-0.21.2
```

Comment:
can you make these changes to main?

Making the changes in a feature branch is actually kinda high-risk, due to the possibility of breaking the bumpenvs flow.

Reply:
Didn't realize how this interacted with bumpenvs, I reverted these changes